### PR TITLE
Tabs: Change the default color of the selected tabs and add activeTabColor

### PR DESCRIPTION
**Pull request by ChengYanJin**
**Component**: Tabs

**Description**:
And add `activeTabColor` property in the component to change at the app level.

Note that:
We need to keep the commit `23e3c8f904f0c4ab9ed77e8dfbcd7112cf4bc195` because it already merged in the Metalk8s
repo

### CODE REVIEW

File: src/lib/components/tabs/Tabs.component.js [hunk 1]
```diff
@@ -33,13 +34,14 @@ const TabItem = styled.div`
...
-  ${props => {
+  ${(props) => {
     return (
       !props.selected &&
       css`
         &:hover {
           cursor: pointer;
-          background-color: ${getTheme(props).primary};
+          background-color: ${props.activeTabColor ||
+          getTheme(props).primaryDark1};
```
Comment (nicolas2bert) [inline src/lib/components/tabs/Tabs.component.js]: Why don't we keep `primary` for our default background color? This change will most likely break scality/zenko-ui.

File: src/lib/components/tabs/Tabs.component.js [hunk 2]
```diff
@@ -50,12 +52,12 @@ const TabItemTitle = styled.p`
...
   padding: ${defaultTheme.padding.base} 0 16.5px;
-  ${props => {
-    const { textPrimary } = getTheme(props);
+  ${(props) => {
+    const { textPrimary, primaryDark1 } = getTheme(props);
     return props.selected
       ? css`
           color: ${textPrimary};
-          background-color: ${getTheme(props).primary};
+          background-color: ${props.activeTabColor || primaryDark1};
```
Comment (nicolas2bert) [inline src/lib/components/tabs/Tabs.component.js]: Same comment here.

File: src/lib/components/tabs/Tabs.component.js (hunk 1 above)
Comment (ChengYanJin) [reply src/lib/components/tabs/Tabs.component.js]: ok. thanks for the review!

I changed back to `primary` as the default background color.
I will set `activeTabColor` in my case.

**Review by nicolas2bert — APPROVED** [review]

**Pull request merged**